app/src/main/res/layout/list_patient_search.xml

```diff
@@ -19,7 +20,7 @@
     android:paddingTop="@dimen/spacing_16">
```

Comment:
As per above comment, you can put this `LinearLayout` in another XML file and inflate it from the `PatientSearchResultView`.

Reply:
Done.